### PR TITLE
bazel/build/ci: add terraform fmt check

### PR DESCRIPTION
**Pull request by cpanato**
fixes: https://jira.coreos.com/browse/INST-879

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -166,8 +166,9 @@ pipeline {
 
                 withDockerContainer(tectonicBazelImage) {
                   sh"""#!/bin/bash -ex
+                    bazel test terraform_fmt --test_output=all
```
Comment (mxinden) [inline Jenkinsfile]: I think you can combine the two test targets into one, like the `bazel build tarball tests/smoke` below, right?

Comment (squat) [reply Jenkinsfile]: We can but if we separate them then we can view them as separated steps in blue ocean I think

Comment (mxinden) [reply Jenkinsfile]: @squat Ah, sounds like a good idea. As far as I know, you need to use a separate `sh` step for each one then.

Comment (cpanato) [reply Jenkinsfile]: I will split in separate `sh`

Comment (cpanato) [reply Jenkinsfile]: done

**Comment by squat** [comment]
hmm not sure about the need for different sh blocks. I’ve seen that each of the cp calls result in separate blocks in blue ocean

**Pull request merged**